### PR TITLE
Added ventricle linux logo

### PR DESCRIPTION
**Pull request by TruncatedDinosour**
## Description

Added ventricle linux ASCII logo
image

## Features
Added ventricle linux ASCII logo

## Issues
None known.

## TODO
N/A

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -11389,7 +11412,7 @@ oMMMMmMMMMNds:.+MMMmmMMN/.-odNMMMMmMMMM+
              .odNm+  /dNms.
 EOF
                 ;;
-
+
```
Comment (dylanaraps) [inline neofetch:11415]: Remove this change.

**Comment by TruncatedDinosour** [comment]
Remove the requested change

File: neofetch [hunk 2]
```diff
@@ -11389,7 +11412,6 @@ oMMMMmMMMMNds:.+MMMmmMMN/.-odNMMMMmMMMM+
              .odNm+  /dNms.
 EOF
                 ;;
-
```
Comment (dylanaraps) [inline neofetch:11392]: I meant remove it from the diff, not the file.

Comment (dylanaraps) [reply neofetch:11392]: It's unrelated to this PR.

Comment (TruncatedDinosour) [reply neofetch:11392]: Sorry, I am confused

**Pull request closed without merging**